components/eam/src/chemistry/yaml/mo_setrxt/setrxt_output_ts_1405.py

```diff
@@ -0,0 +1,22 @@
+# Data at [lat=   39.0553303768561      , lon=   262.904774388088      , k=          54 , time step=        1405 ] 
+# This file was generated by E3SM.
+
+from math import nan as nan, inf as inf
+
+# Object is just a dynamic container that stores input/output data.
+class Object(object):
+    pass
+# Settings are stored here.
+settings = Object()
+# Input is stored here.
+input = Object()
+input.dt = [    0.00, ]
+input.ncol=[[         4],]
+input.temp=[[0.2527505422E+003],]
+input.rate=[[              NaN,              NaN,              NaN,              NaN,              NaN,              NaN,              NaN,],]
```

Comment:
same question on the NaNs.

Reply:
We should remove NaNs. I will have issue in python scripts.

Reply:
If we're going to use NaNs in Python scripts, they should be spelled `nan`, not `NaN`. But I agree that we should probably be more careful about using NaNs to represent values.